src/index.css

```diff
@@ -145,74 +195,62 @@ a:hover {
   box-shadow: 0 5px 5px 0 var(--shadow), 0 4px 0px 0 var(--main-shadow);
 }
 
-.nav-link .fa-github {
-  width: 2.1em;
-  height: 2.1em;
-}
-
-.navbar-light .navbar-nav .nav-link:hover {
-  border-bottom: solid var(--secondary-accent) .17em;
-}
+/* ############################# */
+/* #PAGE                       # */
+/* ############################# */
 
-.navbar-light .navbar-nav .nav-item:last-child .nav-link {
-  margin-right: 0;
-}
-
-.navbar-brand {
-  content: ' ';
-  background-image: url("/img/DraCor-white.svg");
-  background-repeat: no-repeat;
-  width: 4em;
-  height: 4em;
-  margin: 1em 0 1em 2em;
-}
-
-.nav-link span {
-  display: none;
-}
-
-.navbar-collapse .ml-auto, .navbar-collapse .mx-auto {
-    margin-right: 2em;
-    margin-left: 0 !important;
+.content {
+  position: relative;
 }
 
-.navbar-collapse .git-icon {
-  margin: 1em 2em 1em 0;
+.content::before {
+  content: '';
+  background: var(--main);
+  height: 10em;
+  top: -10em;
+  width: 100%;
+  display: block;
+  position: absolute;
 }
 
-.mr-auto, .mx-auto {
-    margin: 0 auto;
+.container-fluid {
+  padding: 0;
```

Comment:
`container-fluid` is a bootstrap class. Overriding the padding like this looks a bit scary to me because it messes with the bootstrap grid system. We should either not use container fluid here or find a bootstrap conformant way to control the padding.